simvue/client.py

```diff
@@ -468,3 +484,36 @@ def get_events(self, run, filter=None, start=0, num=0):
             return response.json()['data']
 
         raise Exception(response.text)
+    
+    def get_alerts(self, run, triggered_only = True, names_only = True):
+        """_summary_
+
+        Parameters
+        ----------
+        run : str
+            The ID of the run to find alerts for
+        critical_only : bool, optional
+            Whether to only return details about alerts which are currently critical, by default True
+        names_only: bool, optional
+            Whether to only return the names of the alerts (otherwise return the full details of the alerts), by default True
+        """
+        response = requests.get(f"{self._url}/api/runs/{run}", headers=self._headers)
+
+        if response.status_code == 404:
+            if 'detail' in response.json():
+                if response.json()['detail'] == 'run does not exist':
+                    raise Exception('Run does not exist')
```

Comment:
Please avoid raising raw Exceptions (I know Simvue has this in places - I have some changes in to remove them).

I recommend `RuntimeError` here 🙂